### PR TITLE
scan: clean up output from scan loop to remove skipped scans/families.

### PR DESCRIPTION
**Pull request by jacobhaven**

### CODE REVIEW

File: scan/scan_common.go [hunk 1]
```diff
@@ -190,7 +193,7 @@ func (fs FamilySet) RunScans(host, family, scanner string, dur time.Duration) (m
 	select {
 	case <-timeout:
 		log.Warningf("scan: %s timed out after %v", host, dur)
-		return nil, nil
+		return familyResults, nil
```
Comment (jacobhaven) [inline scan/scan_common.go]: We might as well return the partially completed family results here.

**Comment by jacobhaven** [comment]
@taravancil You might want to look at this and make sure it doesn't break any of your concurrent-scan work.

**Comment by taravancil** [comment]
@jacobhaven did you see the [changes](https://github.com/cloudflare/cfssl/pull/278/files) I pushed this morning? Individual scans run concurrently now and the results are sent on a channel as they finish. The updates aren't compatible with your changes, so lmk what you think about my approach.

Also, the build is failing for some reason, so I need to look into that...

**Comment by jacobhaven** [comment]
@taravancil That will probably be the more general solution. This was just to remove kruft from the current output.

**Comment by taravancil** [comment]
LGTM!

**Comment by kisom** [comment]
LGTM.

**Pull request merged**